aws_lambda_builders/runtime.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
Can we also call this `validate.py` instead of runtime? It is more representative of what this file does.

Reply:
Done.